avm/client/package.json

```
@@ -20,7 +20,7 @@
     },
     "devDependencies": {
         "@types/jest": "^26.0.23",
-        "jest": "^26.6.3",
+        "jest": "^26.5.0",
```

I believe this wouldn't have any effect because `^26.5.0` means `>=26.5.0 && <27.0.0`. If you want to pin `26.5.0`, drop `^`:
```suggestion
        "jest": "26.5.0",
```